internal/services/bot/bot_channel_web_chat_resource.go

```diff
@@ -53,16 +54,72 @@ func resourceBotChannelWebChat() *pluginsdk.Resource {
 				ValidateFunc: validate.BotName,
 			},
 
-			"site_names": {
+			"site": {
 				Type:     pluginsdk.TypeSet,
-				Required: true,
-				Elem: &pluginsdk.Schema{
-					Type:         pluginsdk.TypeString,
-					ValidateFunc: validation.StringIsNotEmpty,
+				Optional: true,
+				Computed: !features.FourPointOhBeta(),
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"name": {
+							Type:         pluginsdk.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+
+						"application_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
+						},
+
+						"block_user_upload_enabled": {
```

Comment:
This name is confusing - this has to be renamed to 
```suggestion
						"user_upload_enabled": {
```

Reply:
Updated

Reply:
I renamed it to "user_upload_enabled".